apis/livepeer/livepeer.go

```diff
@@ -64,9 +64,10 @@ type (
 		} `json:"servers,omitempty"`
 	}
 
-	createStreamReq struct {
-		Name    string   `json:"name,omitempty"`
-		Presets []string `json:"presets,omitempty"`
+	CreateStreamReq struct {
+		Name     string   `json:"name,omitempty"`
+		ParentID string   `json:"parentId,omitempty"`
```

Comment:
How come ParentID wasn't here before?

Reply:
So the `parentID` was actually only added to the path of the creation request like this: 
https://github.com/livepeer/stream-tester/pull/129/files#diff-df99bcc770f1faed4b227511eaadf4caeab0940378f2046e2e2d526176a6acf9L414

It is not a problem to add the field in the request body though, since when we are setting it with the same value as the value set in the path, and also because in the end it will be just ignored: the child stream will have its `parentId` field [coming from the parent stream](https://github.com/livepeer/livepeer-com/blob/dffc577067cfc88c2c32876f2ff28491c77494c8/packages/api/src/controllers/stream.ts#L721), which is just fetched from the [`streamId` (parent ID) param in the path](https://github.com/livepeer/livepeer-com/blob/dffc577067cfc88c2c32876f2ff28491c77494c8/packages/api/src/controllers/stream.ts#L625).

So I am OK with keeping this field in the `POST` request, also because it is also abiding to the `stream` object schema anyway, so even if we do that validation sometime it will continue to be valid.

If you think it would be better no to include this field in the request payload though, it is really easy to do so as well. I can just change the `json` tag here to be `json:"-"` instead. WDYT?

Reply:
I don't see a prob with including the field in the payload, just wanted to confirm what it is for.